src/Tools/Driver/Curl.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace OSMAviation\Recaptcha\Tools\Driver;
+
+class Curl extends AbstractDriver implements DriverInterface
```

Comment:
Do we even need this now?

Reply:
Despite that Guzzle is a part of Laravel 7 I decided to keep curl, just in case. But if you think it's redundant I can remove all that of course and use the HTTP Client only :-).

Reply:
I think it is redundant, as we've got L7 in our toolbelt, and I want to reduce unused code :)